README.md

```diff
@@ -1 +1,201 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello microverse project <a name="about-project"></a>
+
+> This is my first microverse project. I created using HTML and CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- LINTERS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="http://www.css.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Web browser.
+2. Code Editor.
+3. Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/gitabiy/Hello-Microverse.git
+```
+
+### Install
+
+Run this command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run tests
+
+Coming soon
+
+### Deployment
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Abiy Berhe
+
+- GitHub: [@githubhandle] (https://github.com/gitabiy)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abiy-berhe-94288021a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add Header.
+- [ ] Add Footer.
+- [ ] Add more styles.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give it a star ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] **Good job on using the microverse Readme template 👍** However,

    I think it would be better to provide a  valid **LICENSE** link in your **README**. As open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 

    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

Reply:
Hi Tuyishimire,

I have changed the link file and it works now.
Thank you for the review and feedback.